packages/node-core/src/indexer/fetch.service.spec.ts

```diff
@@ -349,7 +378,7 @@ describe('Fetch Service', () => {
 
   it('enqueues modulo blocks WITHOUT dictionary', async () => {
     // Set modulos to every 3rd block. We only have 1 data source
-    fetchService.modulos = [3];
+    (fetchService as any).getModulos = () => [3];
```

Comment:
Because we have a testing implementation of FetchService, we should add functionality to that rather than doing `any` casts